docs/source/app-dev/bindings-java/quickstart/template-root/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>spark-core</artifactId>
             <version>2.7.2</version>
         </dependency>
+        <dependency>
```

Comment:
It looks like this is no longer in the classpath by virtual of being pulled in as a transitive dependency. Seems sensible to add it as a direct dep anyway if we rely on it.